cypress/tests/integration/DataAvailabilityStatements.spec.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file cypress/tests/integration/DataAvailabilityStatements.spec.js
+*/
+
+describe('DataAvailabilityStatements', function () {
+	var statement = 'This is an example of a data availability statement';
+
+	it('Enables Data Availability Statements as submission metadata', function () {
+		cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/management/settings/workflow#submission/metadata');
+		cy.wait(500);
+        cy.contains('Enable Data Availability Statement metadata').find('[type="checkbox"]').check()
+        cy.wait(500);
+		cy.get('#metadata button').contains('Save').click();
+		cy.get('#metadata [role="status"]').contains('Saved');
+	});
+    
+    it('Adds a statement to a submission', function () {
+        cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/workflow/index/14/1');
```

Comment:
It's best not to embed database IDs in the test. The best approach is to have the test find the content e.g. via the editorial interface by title or author name, then store its ID in an [alias](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases) for the testing code to use.

Admittedly, sometimes we do embed database IDs in our own tests; if you do continue that way, it's best to at least pull this out into a constant near the top of the file.

Reply:
Sorry for this smelly code, done removing the database IDs from the specs